apps/docs/components/Playground.js

```diff
@@ -104,7 +133,8 @@ export default function Playground() {
                 theme: 'material-darker',
                 lineNumbers: true,
               }}
-              value={files.src.directory['app.jsx'].file.contents}
+              value={code}
+              onChange={(editor, data, newCode) => handleCodeChange(newCode)}
             />
             {url ? (
```

Comment:
Doing this causes the iFrame to remount after the update. This forces it to pull the latest changes.

```suggestion
            {url && !isUpdating ? (
```